plutus-core/plutus-ir/test/transform/inline/letOverAppType2

```diff
@@ -1,53 +0,0 @@
-{- 
```

Comment:
why is this gone?

Reply:
As I mentioned this is a duplicate to `letOverAppType`.